spec/all_pages_spec.rb

```diff
@@ -8,6 +8,10 @@ def redirect_page?(path)
   File.read(file_at(path)).include?('<meta http-equiv="refresh"')
 end
 
+def external_link?(uri)
+  uri.scheme && !/^(www\.)?#{URI(SITE_URL).host}$/.match(uri.host)
```

Comment:
what if we made a constant like `SITE_URI = URI(SITE_URL)` so we don't have to parse each time?

```suggestion
  uri.scheme && !/^(www\.)?#{SITE_URI.host}$/.match(uri.host)
```

Reply:
> what if we made a constant like `SITE_URI = URI(SITE_URL)` so we don't have to parse each time?

Yeah, good idea. Updated in 6afea3d.